### PR TITLE
DDF-1965 Fix bugs in conditional logic

### PR DESCRIPTION
**Pull request by brendan-hofmann**

### CODE REVIEW

File: support-hubot/hubot-bamboo-prbuild/index.coffee [hunk 1]
```diff
@@ -35,10 +35,9 @@ module.exports = (robot) ->
 
     # Determines whether a pr build or master build is needed.
     getPlanName = (eventPayload) ->
-        closed = eventPayload.action is "closed"
+        prClosed = eventPayload.action is "closed"
         merged = eventPayload.pull_request.merged
```
Comment (lessarderic) [inline support-hubot/hubot-bamboo-prbuild/index.coffee]: To be consistent, do you think we should call this one `prMerged`?

File: support-hubot/hubot-bamboo-prbuild/index.coffee [hunk 2]
```diff
@@ -225,14 +224,14 @@ module.exports = (robot) ->
         gitHubUrl = eventPayload.repository.url
         prNumber = eventPayload.number
         statusUrl = eventPayload.pull_request.statuses_url
-        prPlanName = getPlanName(eventPayload)
+        planName = getPlanName(eventPayload)
```
Comment (lessarderic) [inline support-hubot/hubot-bamboo-prbuild/index.coffee]: On top of this, should we also rename the two "constants" at the top of the file `PR_PLAN_NAME` and `MASTER_PLAN_NAME` or something?

Comment (brendan-hofmann) [inline support-hubot/hubot-bamboo-prbuild/index.coffee]: In retrospect, that would be a good idea.

File: support-hubot/hubot-bamboo-prbuild/index.coffee (hunk 1 above)
Comment (brendan-hofmann) [inline support-hubot/hubot-bamboo-prbuild/index.coffee]: We could, but `prClosed` is kind of silly too. I was trying to think of a better way to namespace these.

**Pull request merged**